File: nvxapp.server.service/ClientServer-Service/GestionePresenze/Az_AttivitaCompetenzaService/Models/Az_AttivitaCompetenzaModel.cs
using nvxapp.server.service.ClientServer_Service.ModelsBase;

namespace nvxapp.server.service.ClientServer_Service.GestionePresenze.Az_AttivitaCompetenzaService.Models
{
    public class Az_AttivitaCompetenzaModel
    {
        public int Id { get; set; }
        public int IdAz_Attivita { get; set; }
        public string? Descrizione { get; set; }
    }

    public class Az_AttivitaCompetenza_GetAll_InModel { }

    public class Az_AttivitaCompetenza_GetAll_OutModel : ModelResult
    {
        public List<Az_AttivitaCompetenzaModel> Az_AttivitaCompetenza { get; set; } = new List<Az_AttivitaCompetenzaModel>();
    }
}

File: nvxapp.client/src/nvx/ClientServer-Service/GestionePresenze/Az_AttivitaCompetenza/Models/az-attivita-competenza-model.ts
import { ModelResult } from "../../../ModelsBase/model-result";

export class Az_AttivitaCompetenzaModel {
  public id!: number;
  public idAz_Attivita!: number;
  public descrizione: string;
}

export class Az_AttivitaCompetenza_GetAll_InModel {}

export class Az_AttivitaCompetenza_GetAll_OutModel extends ModelResult {
  public az_AttivitaCompetenza: Az_AttivitaCompetenzaModel[] = [];
}
